packages/button/src/vaadin-button-mixin.js

```diff
@@ -56,6 +56,10 @@ export const ButtonMixin = (superClass) =>
       }
     }
 
+    __modifierKey(event) {
+      return event.altKey || event.shiftKey || event.ctrlKey || event.metaKey;
+    }
```

Comment:
I'd prefer to inline this check to `_onKeyDown` e.g. as it is done in this mixin:

https://github.com/vaadin/web-components/blob/9e8498d56d67904b1e030ec597a5ab2c30c94613/packages/a11y-base/src/list-mixin.js#L249-L251

Reply:
Done